src/App.js

```diff
@@ -1,23 +1,81 @@
-import styled from 'styled-components';
+// import Heading from './components/Heading';
 import Dashboard from './pages/Dashboard';
-import useFetch from './hooks/useFetch';
+import { useState, useEffect } from 'react';
 
-function App() {
-  const { issues, loading, error } = useFetch(
-    'https://api.github.com/repos/reactjs/reactjs.org/issues'
-  );
+export default function App() {
+  const [issues, setIssues] = useState([]);
+  const [loading, setLoading] = useState(null);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    GetFetch('https://api.github.com/repos/reactjs/reactjs.org/issues');
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
 
   return (
-    <Container className="App">
-      <Dashboard issues={issues} loading={loading} error={error} />
-    </Container>
+    <>
```

Comment:
here you only have the Dashboard so you do not need the <> around I think

Reply:
thanks i fixed it